@xen-orchestra/web/src/pages/vm/[id]/system.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="system">
+    <div class="column">
+      <VmGeneralInfo :vm />
+      <VmSystemNetworking :vm />
+      <VmStorageConfig :vm />
+      <VmResource :vm />
+    </div>
+    <div class="column">
+      <VmVirtualisationAndBoot :vm />
+      <VmManagement :vm />
+      <VmGraphics :vm />
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import VmGeneralInfo from '@/components/vm/VmGeneralInfo.vue'
+import VmGraphics from '@/components/vm/VmGraphics.vue'
+import VmManagement from '@/components/vm/VmManagement.vue'
+import VmResource from '@/components/vm/VmResource.vue'
+import VmStorageConfig from '@/components/vm/VmStorageConfig.vue'
+import VmSystemNetworking from '@/components/vm/VmSystemNetworking.vue'
+import VmVirtualisationAndBoot from '@/components/vm/VmVirtualisationAndBoot.vue'
+import type { XoVm } from '@/types/xo/vm.type'
+
+defineProps<{ vm: XoVm }>()
+</script>
+
+<style scoped lang="postcss">
+.system {
+  display: flex;
+  gap: 0.8rem;
+  padding: 0.8rem;
+  width: 100%;
+
+  .column {
+    display: flex;
+    flex-direction: column;
+    gap: 0.8rem;
+  }
+
+  @media not (--mobile) {
+    flex-direction: row;
+
+    .column {
+      width: 50%;
+    }
+  }
+
+  @media (--mobile) {
+    flex-direction: column;
+
+    .column {
+      width: 100%;
+    }
+  }
```

Comment:
While this works, in the project we tend to add a class when we want to change the layout on mobile.
So, in this case, we would have something like this (remember to use the `uiStore` to add the class in the template section):
```suggestion
.system {
  display: flex;
  gap: 0.8rem;
  padding: 0.8rem;
  width: 100%;
  flex-direction: row;

  .column {
    display: flex;
    flex-direction: column;
    gap: 0.8rem;
    width: 50%;
  }

  &.mobile {
    flex-direction: column;

    .column {
      width: 100%;
    }
  }
}
```